.github/ISSUE_TEMPLATE/removing_data_or_service.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Removing Data From Anywhere
```

Comment:
i think we shoudl replace the existing issue, will you rename this to, `deprecate-sgid-dataset-or-web-application.md`

Reply:
Yeah, that makes sense, I'll switch it over.